src/pages/index.tsx

```diff
@@ -1,3 +1,20 @@
+import Navigation from 'src/components/Navigation';
+import HeroSection from 'src/components/HeroSection';
```

Comment:
Let's use `@/components/` after we merge `tsconfig` will handle that
```suggestion
import Navigation from '@/components/Navigation';
import HeroSection from '@/components/HeroSection';
```